packages/odyssey-react-mui/src/Dialog.tsx

```diff
@@ -139,9 +139,17 @@ const Dialog = ({
   ].filter(Boolean);
 
   return (
-    <MuiDialog data-se={testId} open={isOpen} onClose={onClose}>
-      <DialogTitle translate={translate}>
-        {title}
+    <MuiDialog
+      data-se={testId}
+      open={isOpen}
+      onClose={onClose}
+      aria-labelledby="ods-dialog-title"
+    >
+      <DialogTitle
+        id="" // mui automatically set id based on aria-labelledby from MuiDialog, explicitly unset it to use id from span
```

Comment:
I had trouble understanding this comment, so I rewrote it in a way that should make sense for people unfamiliar with the issue. Hopefully I didn't change it to something wrong.

```suggestion
        id="" // We need to explicitly unset `id` for MUI to automatically set it based on the `aria-labelledby` prop passed to `MuiDialog`.
```

I'm not quite sure what's going on here. You say you're unsetting `id`, but you're setting it to `""`. Unsetting it should be removing the `id` prop or using `{undefined}` right?

Why are we using a null string? I would include that as part of the comment or change the code to match. I wanna make it easier on the next dev that passes by this code.

Reply:
updated to `ods-dialog-title__container`